From d8bc967ce30834f6b22c9bb36963e70e99421a17 Mon Sep 17 00:00:00 2001
From: billy4479 <giachi.ellero@gmail.com>
Date: Fri, 2 Feb 2024 16:38:01 +0100
Subject: [PATCH 2/4] Use DevMain

---
 build.gradle.kts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/build.gradle.kts b/build.gradle.kts
index 58ac8a6..8ca481b 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -43,14 +43,14 @@ dependencies {
 }
 
 application {
-	mainClass.set("link.infra.packwiz.installer.RequiresBootstrap")
+	mainClass.set("link.infra.packwiz.installer.DevMainKt")
 }
 
 version = "###VERSION###"
 
 tasks.jar {
 	manifest {
-		attributes["Main-Class"] = "link.infra.packwiz.installer.RequiresBootstrap"
+		attributes["Main-Class"] = "link.infra.packwiz.installer.DevMainKt"
 		attributes["Implementation-Version"] = project.version
 	}
 }
-- 
2.43.0

