From 2bea454e42be192bfee05c0015ef154cfc4c7849 Mon Sep 17 00:00:00 2001
From: billy4479 <giachi.ellero@gmail.com>
Date: Thu, 1 Feb 2024 13:18:27 +0100
Subject: [PATCH 4/4] Add resolve task

---
 build.gradle.kts | 19 +++++++++++++++++++
 1 file changed, 19 insertions(+)

diff --git a/build.gradle.kts b/build.gradle.kts
index 8ca481b..e261bc8 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -202,3 +202,22 @@ if (project.hasProperty("bunnycdn.token")) {
 	}
 }
 
+tasks.register("resolveDependencies") {
+    doLast {
+        project.rootProject.allprojects.forEach { subProject ->
+            subProject.buildscript.configurations.forEach { configuration ->
+                resolveConfiguration(subProject, configuration, "buildscript config '${configuration.name}'")
+            }
+            subProject.configurations.forEach { configuration ->
+                resolveConfiguration(subProject, configuration, "config '${configuration.name}'")
+            }
+        }
+    }
+}
+
+fun resolveConfiguration(subProject: Project, configuration: Configuration, name: String) {
+    if (configuration.isCanBeResolved) {
+        logger.info("Resolving project {} {}", subProject.name, name)
+        configuration.resolve()
+    }
+}
-- 
2.43.0

